tests/Resources/Page/ODH/AiApps/Rhoam.resource

```diff
@@ -31,3 +35,8 @@ Wait Until RHOAM Installation Is Completed
         Fail    RHOAM didn't get "complete" stage after ${retries} retries. Check the installation..
     END
 
+Verify RHOAM Is Enabled IN RHODS Dashboard
+    Launch Dashboard    ocp_user_name=${TEST_USER.USERNAME}    ocp_user_pw=${TEST_USER.PASSWORD}
+    ...    ocp_user_auth_type=${TEST_USER.AUTH_TYPE}    dashboard_url=${ODH_DASHBOARD_URL}    browser=${BROWSER.NAME}
+    ...    browser_options=${BROWSER.OPTIONS}    $ocp_user_name    $ocp_user_pw    $ocp_user_auth_type    $dashboard_url    $browser    $browser_options
```

Comment:
Are all of these parameters needed here?

Reply:
ops, I'll delete them copy-past error. Thanks